services/src/main/java/cz/muni/fi/pa165/services/UserServiceImpl.java

```diff
@@ -95,6 +124,16 @@ public void update(Users user) {
         userDao.update(user);
 
     }
+    @Override
+    public boolean isAdmin(Users user) {
+        try {
+            return findById(user.getId()).getRole().getName().equals("ADMIN");
```

Comment:
Hrube reseni, ale vlastne proc ne... Jen to prosim te porovnavej s nejakou konstantou a ne se stringem, ktery jsi sem jen tak napsal